include/cantera/kinetics/Arrhenius.h

```diff
@@ -317,9 +314,9 @@ class BlowersMaselRate : public ArrheniusBase
         return m_deltaH_R * GasConstant;
     }
 
-    //! Set enthalpy change without Kinetics object. In most cases, the enthalpy
-    //! change is set during the update method, which, however, requires the
-    //! object to be linked to a Kinetics object.
+    //! Set enthalpy change without a Kinetics object (for testing purposes).
```

Comment:
It may make sense to eliminate this 'bypass' ... at this point, `evaluateFromStruct` always gets a `ReactionData` object (either managed, or created on-the-fly). For the on-the-fly version, it may make sense to implement `eval(T, P, dH)`  together with a `ReactionData::update(T, P, dH)`, and disable the `ReactionData::update(T)` and `ReactionData::update(T, P)`. I don't want to get rid of the evaluators altogether though, as it would make unit testing much harder.